lib/src/sentry.ts

```diff
@@ -31,14 +33,24 @@ export function initSentry(sentry: boolean) {
   });
 
   hub = new Sentry.Hub(client);
+  return getSentryHubWrapper(hub);
+}
 
-  hub.configureScope(function (scope) {
-    scope.setTag('source', '@hCaptcha/loader');
-  });
+export function getSentry(): SentryHub | null {
+  return getSentryHubWrapper(hub);
+}
 
-  return hub;
+function setScopeTag(source: string = '@hCaptcha/loader') {
+  hub?.configureScope(function(scope) {
+    scope.setTag('source', source);
```

Comment:
Shouldn't this be more generic incase you want to set a `key | value`? In this case having two parameters to pass in.

Reply:
yes, changed